libs/types/src/generateKnownType.ts

```diff
@@ -3,9 +3,9 @@ import { GenerateContext } from "./generateContext";
 
 export function generateKnownType(knownType: SchemaKnownType, context: GenerateContext) {
     const typeGenerator = context.typesMap[knownType.type];
-    const outputType = typeGenerator({ typeArguments: knownType.typeArguments, context });
+    const outputType = typeGenerator && typeGenerator({ typeArguments: knownType.typeArguments, context });
```

Comment:
Why the change? Something has changed here? Shouldn't the `typesMap` be also changed to reflect the types? It seems not really connected to the rest of the PR.

Also it should be like this I guess:

```suggestion
    const outputType = typeGenerator?.({ typeArguments: knownType.typeArguments, context });
```

Reply:
It's just a sanity check against invalid inputs. This path shouldn't be exercised unless the server generator emits a type that this one doesn't know about.

Tbh I could just drop this commit, all it really does is make unexpected incompatibilities easier to detect.